projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -612,7 +612,7 @@ function render_for_website( $data, $classes, $styles ) {
 	$blog_id           = \Jetpack_Options::get_option( 'id' );
 	$widget_id_suffix  = Jetpack_Subscriptions_Widget::$instance_count > 1 ? '-' . Jetpack_Subscriptions_Widget::$instance_count : '';
 	$form_id           = 'subscribe-blog' . $widget_id_suffix;
-	$form_url          = defined( 'SUBSCRIBE_BLOG_URL' ) ? SUBSCRIBE_BLOG_URL : '#';
+	$form_url          = 'https://wordpress.com/email-subscriptions';
```

Comment:
Fine for first iteration, but there's a way to get specific site's subscription page vs just the generic list of all subscriptions.

Please test how both logged-in and logged-out experiences feel.